08-projects.Rmd

```diff
@@ -10014,7 +10004,7 @@ Make sure to replace "\<title id here\>" with the `title_id` of your favorite sh
 
 #### Dataset: 
 
-The following questions will use the dataset found in Scholar:
+The following questions will use PLOTSNAP.csv from the data folder found in Scholar:
```

Comment:
Ah, dear @kevinamstutz , could you help me adding TREE.csv here? I forgot to type it in in my pull request version... Thanks!

Reply:
Yep! No problem!